src/sources/soundsourceffmpeg.cpp

```diff
@@ -485,9 +477,9 @@ long SoundSourceFFmpeg::seek(long filepos) {
         // Try to find some jump point near to
         // where we are located so we don't needed
         // to try guess it
-        if (filepos >= SOUNDSOURCEFFMPEG_POSDISTANCE) {
+        if (frameIndex >= AUDIOSOURCEFFMPEG_POSDISTANCE) {
             for (i = 0; i < m_SJumpPoints.size(); i ++) {
-                if (m_SJumpPoints[i]->startByte >= (unsigned long) filepos && i > 2) {
+                if (m_SJumpPoints[i]->startByte >= (unsigned long) frameIndex && i > 2) {
```

Comment:
Is startByte actually a Byte or a Sample or a Frame? 
What is the 2? Is it still required?